public/app/types/unified-alerting.ts

```diff
@@ -111,6 +111,13 @@ export interface RuleWithLocation<T = RulerRuleDTO> {
   rule: T;
 }
 
+export interface CombinedRuleWithLocation {
```

Comment:
Should we extend `CombinedRule` interface instead of nesting `CombinedRule` as a field? 
Due to nesting, we need to write `rule.rule.promRule` which is a bit weird (actually, `rule.promRule` is already a bit weird 😄)